<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="sysadm-Latitude-3480" ignored="0" name="Account_Service_peoplecontroller_GetuserById" tests="2" failures="1" timestamp="04 May 2020 05:15:16 GMT" time="0.721" errors="0">
  <testcase name="peoplecontroller_getuserbyid" time="0.341" classname="accountsAPI.Account_Service_Peoplecontroller_GetuserById">
    <skipped/>
  </testcase> <!-- peoplecontroller_getuserbyid -->
  <testcase name="peoplecontroller_getuserbyid" time="0.306" classname="accountsAPI.Account_Service_Peoplecontroller_GetuserById">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:48)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at accountsAPI.AccountsCommon_API.validation(AccountsCommon_API.java:1121)
at accountsAPI.Account_Service_Peoplecontroller_GetuserById.peoplecontroller_getuserbyid(Account_Service_Peoplecontroller_GetuserById.java:17)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- peoplecontroller_getuserbyid -->
</testsuite> <!-- Account_Service_peoplecontroller_GetuserById -->
